packages/rulesets/generated/spectral/az-dataplane.js

```diff
@@ -1942,6 +1946,7 @@ const ruleset = {
     rules: {
         AdditionalPropertiesAndProperties: {
             description: "Don't specify additionalProperties as a sibling of properties.",
+            ignoreForTypespec: true,
```

Comment:
I'm not sure if we have a complete guard against this happening in TSP.

Reply:
we do `@azure-tools/typespec-azure-core/bad-record` https://azure.github.io/typespec-azure/playground?c=LyoqIFdpdGggZG9jICovCm1vZGVsIEZvbyBpcyBSZWNvcmQ8c3RyaW5nPiB7CiAg0DAgIG5hbWU6IMYkOwp9Cg%3D%3D&e=%40azure-tools%2Ftypespec-autorest&options=%7B%22linterRuleSet%22%3A%7B%22extends%22%3A%5B%22%40azure-tools%2Ftypespec-azure-core%2Fall%22%5D%7D%7D

Reply:
Gah, I was using a [stale playground link](https://cadlplayground.z22.web.core.windows.net/cadl-azure/?c=bW9kZWwgRG9lc0l0V29yayBleHRlbmRzIFJlY29yZDxzdHJpbmc%2BIHsKICBiYXI6IMYROwp9&e=%40azure-tools%2Ftypespec-autorest&options=%7B%7D) when I tested.

Reply:
That links works too but the linters are not enabled by default. You have to select them